CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 0.4.3
```

Comment:
Probably want to bump the major version as this is a breaking change in the config.